src/pages/ResearcherProfile.js

```diff
@@ -79,72 +46,28 @@ export default function ResearcherProfile(props) {
   }, [props.history]);
 
   useEffect(() => {
-    const isValidState = (value) => {
-      const stateSelected = (!isNil(value) && !isEmpty(value));
-      const inUS = (profile.country === 'United States of America' || profile.country === '');
-      if (inUS && stateSelected) {
-        return true;
-      }
-      return !inUS;
-    };
-
-    let incompletes = [];
-
-    if (!isValid(profile.profileName)) {
-      incompletes.push('Name');
-    }
-
-    if (!isValid(profile.academicEmail) || !isEmailAddress(profile.academicEmail)) {
-      incompletes.push('Email Address');
-    }
-
-    if (!isNumber(profile.institutionId) || profile.institutionId === 0) {
-      incompletes.push('Institution');
-    }
-
-    if (!isValid(profile.address1)) {
-      incompletes.push('Address');
-    }
-
-    if (!isValid(profile.city)) {
-      incompletes.push('City');
-    }
-
-    if (!isValidState(profile.state)) {
-      incompletes.push('State');
-    }
-
-    if (!isValid(profile.country)) {
-      incompletes.push('Country');
-    }
-
-    if (!isValid(profile.zipcode)) {
-      incompletes.push('Zip/Postal Code');
-    }
-
-    if (profile.isThePI === null || profile.isThePI === '' || (profile.isThePI === 'false' && profile.havePI === '')) {
-      incompletes.push('Principal Investigator Information');
-    }
-
-    if (profile.isThePI === 'false' && profile.havePI === 'true') {
-      if (!isValid(profile.piName)) {
-        incompletes.push('Principal Investigator Name');
-      }
-      if (!isValid(profile.piEmail) || !isEmailAddress(profile.piEmail)) {
-        incompletes.push('Principal Investigator Email');
-      }
+    if (profile.institutionId) {
+      Institution.getById(profile.institutionId).then((institution) => {
+        if (!institution) {
+          return;
+        }
+
+        setSigningOfficialList((institution.signingOfficials ? institution.signingOfficials : []));
+      });
     }
+  }, [profile.institutionId]);
 
-    setIncompleteFields(incompletes);
-    setResearcherFieldsComplete(incompletes.length === 0);
-  }, [profile]);
+  const hasInstitution = () => {
+    return (isNumber(profile.institutionId) && profile.institutionId !== 0) || (profile.suggestedInstitution !== undefined && profile.suggestedInstitution !== '');
```

Comment:
heya! do you mean why is institution greyed out? it's probably because that account is a signing official, which means you cannot update your institution. i had to use a second account to test this PR which was just a researcher